dashboard/dashy_endpoints.py

```diff
@@ -101,7 +101,7 @@ def main():
                     "path": DOCS_URL,
                 }
             ],
-            "logo": "https://egi-api.nois3.net/app/uploads/2021/11/egi-logo.svg",
+            "logo": "https://cdn.egi.eu/app/uploads/2021/11/egi-logo.svg",
         },
```

Comment:
If the logo, and other materials (images, CSS, JS and so on,) could be hosted directly into dashy it would remove a dependency on an external endpoint, and avoid the kind of failure we got.

Reply:
good point, re-purposing everything to make the logos local